### PR TITLE
Enable EIP-2930 transactions by default

### LINKED ISSUE #7785: Enable EIP-2930 transactions by default
### Problem

The feature flag for EIP-2930 tx is set to `false` by default.

### Solution

Set `hedera.transaction.eip2930.enabled` to `true`. 

### Alternatives

_No response_

### PR DESCRIPTION
**Pull request by natanasow**
**Description**:

The feature flag for EIP-2930 tx is set to `false` by default. We might set it to `true` due to the mirror node importer changes are already done.

**Related issue(s)**:

Fixes #7785 

**Notes for reviewer**:

**Checklist**

- [x] Documented (Code comments, README, etc.)
- [x] Tested (unit, integration, etc.)

### CODE REVIEW

**Review by stoqnkpL — DISMISSED** [review]
LGTM. 
This will enable the feature on all environments which is what we want to do. 
When opening the cherry-pick PRs to the `release/0.41` branch we need to be careful and only enable the feature for environments which will be running Mirror Node version 86 or above.

**Review by Nana-EC — CHANGES_REQUESTED** [review]
LG.
Can you address this remaining feedback from the last PR

https://github.com/hashgraph/hedera-services/pull/7696#discussion_r1279807424

**Comment by Nana-EC** [comment]
> LGTM. 
> 
> This will enable the feature on all environments which is what we want to do. 
> 
> When opening the cherry-pick PRs to the `release/0.41` branch we need to be careful and only enable the feature for environments which will be running Mirror Node version 86 or above. 

@stoqnkpL and @natanasow mirror node previewnet is 0.86.0-beta1 now so we should be good for the cherry pick for the next services alpha

File: hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/ethereum/EthereumSuite.java [hunk 1]
```diff
@@ -742,6 +743,53 @@ private HapiSpec directTransferWorksForERC20() {
                                                         .withErcFungibleTransferStatus(true)))))));
     }
 
+    HapiSpec transferHbarsViaEip2390TxSuccessfully() {
```
Comment (Nana-EC) [inline hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/ethereum/EthereumSuite.java:746]:
```suggestion
    HapiSpec transferHbarsViaEip2930TxSuccessfully() {
```

Comment (Nana-EC) [inline hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/ethereum/EthereumSuite.java:746]: Can we add a HapiSpec for make a contract function call using 2930 tx type also.
If there's already one present then it's fine

**Review by Nana-EC — CHANGES_REQUESTED** [review]
LG.
Minor suggestion and test request

File: hedera-node/hapi-utils/src/test/java/com/hedera/node/app/hapi/utils/ethereum/EthTxDataTest.java [hunk 2]
```diff
@@ -51,8 +51,21 @@ class EthTxDataTest {
     //  s: '0x249a72e1479b61e00d4f20308577bb63167d71b26138ee5229ca1cb3c49a2e53',
     //  v: 28, accessList: []
     // }
-    static final String RAW_TX_TYPE_1 =
-            "01f87382012a82160c85a54f4c3c00832dc6c094000000000000000000000000000000000000052d8502540be40083123456c001a0abb9e9c510716df2988cf626734ee50dcd9f41d30d638220712b5fe33fe4c816a0249a72e1479b61e00d4f20308577bb63167d71b26138ee5229ca1cb3c49a2e53";
+    static final String RAW_TX_TYPE_1 = "01" + // type
```
[the remark this answers is not in the document]
Comment (lukelee-sl) [reply hedera-node/hapi-utils/src/test/java/com/hedera/node/app/hapi/utils/ethereum/EthTxDataTest.java:54]: looks good.  thanks for doing this.  A nit is to do this to keep the lines consistent?
```suggestion
    static final String RAW_TX_TYPE_1 = "01"  // type
    + "f873" // total length
```

**Review by lukelee-sl — DISMISSED** [review]
LGTM

File: hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/ethereum/EthereumSuite.java (hunk 1 above)
Comment (natanasow) [reply hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/ethereum/EthereumSuite.java]: Added for both contract function `transfer` and contract call `totalSupply`. Feel free to ping me if we need more coverage :rocket:.

**Review by Nana-EC — DISMISSED** [review]
LGTM, thanks

**Review by stoqnkpL — APPROVED** [review]
LGTM!

**Review by Neeharika-Sompalli — APPROVED** [review]

**Pull request merged**